### ISSUE TITLE
[BUG] WireGuard VPN Not Connected Message Appears , Even When It’s On

### ISSUE THREAD

**Issue by PlayfulBowl3512** [SideStore/SideStore#531]
### Describe the bug

I have WireGuard on, Wi-Fi’s the only internet option on my iPad, and it still doesn’t register as enabled! I don’t know if it’s because I installed the configuration before I installed SideStore or something else!
![IMG_1487](https://github.com/SideStore/SideStore/assets/100943240/e03973f7-dc4f-47d3-b83d-4c9616136e5a)

### Instructions to reproduce

I don’t know how it happened! I just did step 3 — install configuration and VPN — before anything else! So first, I installed the WireGuard app and the SideStore configuration file and turned on the VPN, then got the SideStore, iTunes, and iCloud clients for my Windows 10 PC, and finally, installed the SideStore app to my iPad and enabled both the developer configuration and developer mode.

### What version of SideStore are you using?

0.5.2

**Comment by nythepegasus** [comment]
Hello! This is a duplicate of many other issues like this, have you remade your pairing file and ensured you sideloaded SideStore using AltServer? **SideServer installs don't let you choose a new pairing file!**

**Comment by PlayfulBowl3512** [comment]
> Hello! This is a duplicate of many other issues like this, have you remade your pairing file and ensured you sideloaded SideStore using AltServer? **SideServer installs don't let you choose a new pairing file!**

I used SideServer. 😰 Why does the official option not work!?

**Comment by nythepegasus** [comment]
> I used SideServer. 😰 Why does the official option not work!?

Currently SideServer signs SideStore incorrectly and embeds the pairing file making it impossible to change. There's a developer in the community working on a replacement at https://github.com/Dadoum/Sideloader that will get rid of SideServer entirely. We don't want to rush him, and our wiki has the AltServer method listed, though once Sideloader is done we will be replacing everything with that.

**Comment by xhfpsxm** [comment]
> Hello! This is a duplicate of many other issues like this, have you remade your pairing file and ensured you sideloaded SideStore using AltServer? **SideServer installs don't let you choose a new pairing file!**

I am still getting that error. I installed version 0.5.4 through altserver. And I remade pairing file. Please tell me what I should do.

**Comment by riansio** [comment]
I have the same problem too!

**Comment by junepark678** [comment]
can't repro
